lib/chef/resource/cron_d.rb

```diff
@@ -162,7 +163,7 @@ def self.validate_dow(spec)
 
       # warn if someone passes the deprecated cookbook property
       def after_created
-        raise ArgumentError, "The 'cookbook' property for the cron_d resource is no longer supported now that this resource ships in Chef itself." if cookbook
+        raise ArgumentError, "The 'cookbook' property for the cron_d resource is no longer supported now that it ships as a core resource." if cookbook
```

Comment:
core resource in #{Chef::Dist::PRODUCT}

Reply:
actually I removed that at @robbkidd 's request. Not opposed to adding it back.

Reply:
I was thinking that it will be easier if we reduce the number of references. Whether it's "Chef cron_d resource" or "CINC cron_d resource," it's the _same_ resource.